test/__mocks__/data/tiktok_ads/proxy_response.json

```diff
@@ -27,6 +27,14 @@
         "message": "Too many requests. Please retry in some time."
       },
       "status": 200
+    },
+    "502-BadGateway": {
+      "data": "<html>\r\n<head><title>502 Bad Gateway</title></head>\r\n<body bgcolor=\"white\">\r\n<center><h1>502 Bad Gateway</h1></center>\r\n<hr><center>nginx</center>\r\n</body>\r\n</html>\r\n",
```

Comment:
possible to do contains match instead?

Reply:
Sorry, I didn't understand. Can you let me know what contains match are we talking about ? Which value should be checked with contains ?!

Reply:
> Sorry, I didn't understand. Can you let me know what contains match are we talking about ? Which value should be checked with contains ?!

Instead of exact match, can we try contains match

Reply:
>502-BadGateway

Are you talking about the above key matching ? or is it something else ?

Reply:
As discussed we will handle this in a common way later. So for now, resolving this comment.